vocabularies/data-access-rights.ttl

```diff
@@ -0,0 +1,56 @@
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+@prefix da: <http://linked.data.gov.au/def/data-access-rights/> .
+
+<http://linked.data.gov.au/def/data-access-rights> a owl:Ontology .
+
+da:conceptScheme a skos:ConceptScheme ;
+    skos:prefLabel "Data Access Rights"@en ;
+    dct:description "Data access rights control how users and systems access a data resource."@en ;
+    skos:historyNote "This vocabulary is taken from the COAR Controlled Vocabularies Interest Group (http://vocabularies.coar-repositories.org/documentation/access_rights/) but is redelivered as that vocabulary isn't well presented online."@en ;
+    dct:source <http://vocabularies.coar-repositories.org/documentation/access_rights/> ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:created "2019-04-03"^^xsd:dateTime ;
+    dct:modified "2019-09-10"^^xsd:dateTime .
+
+da:embargoed a skos:Concept ;
+    skos:prefLabel "Embargoed access"@en ;
+    skos:definition "Embargoed access refers to a resource that is metadata only access until released for open access on a certain date. Embargoes can be required by regulation or policies, or set by the author."@en ;
+    skos:inScheme da:conceptScheme ;
+    skos:topConceptOf da:conceptScheme .
+
+da:metadata-only a skos:Concept ;
+    skos:prefLabel "Metadata only access"@en ;
+    skos:definition "Metadata only access refers to a resource in which access is limited to metadata only. The metadata is open data but acccess to the resource itself is controlled. Access to the resource requires granting of access rights."@en ;
```

Comment:
Metadata only access refers to a resource in which access is limited to metadata only i.e. only metadata is accessible prior to a specified open-file date for the resource. Access to the resource requires granting of access rights.

_rewording and remove typo from access_